README.md

```diff
@@ -8,16 +8,16 @@ This project implements a field-based call graph construction algorithm for Java
 ## Get Started
 ```
 npm install
-node src/main.js -h # for a list of command line arguments
+node javascript-call-graph -h # for a list of command line arguments
```

Comment:
Like the idea, but can we choose a shorter name? Having to type javascript-call-graph every time too long:)

Reply:
Of course, I create a new patch with "jcg" name :) The javascript-call-graph really long :)